sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Ensure unique `replyTo` when new link is needed in ManagementClient. (PR #24509)[https://github.com/Azure/azure-sdk-for-js/pull/24509]
```

Comment:
damn! great find :)

Reply:
How did you find this?!

Reply:
I added a live test to test MessageSizeExceeded error earlier.  After that some other tests become flaky and timed out some times.